docs/install.md

```diff
@@ -0,0 +1,54 @@
+# Installing Hive
+
+## Prerequisites
+
+* [kustomize](https://github.com/kubernetes-sigs/kustomize#kustomize)
+* [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/)
+
+## Deployment Options
+
+Hive contains an operator which is responsible for handling deployment logic for the rest of the components. In the near future this operator may be installable via OLM.
```

Comment:
Is the OLM bit still true?

Reply:
OLM stuff did not work for me. We need to fix that, but that's outside of the scope of this PR IMO.